File: task-03/Python/Sub-Task4.py
with open('input.txt', 'r') as file:
    n = int(file.read().strip())

with open('output.txt', 'w') as file:
    for i in range(n):
        file.write(" " * (n - i - 1) + "*" * (2 * i + 1) + "\n")
    for i in range(n-2, -1, -1):
        file.write(" " * (n - i - 1) + "*" * (2 * i + 1) + "\n")

File: task-03/Elixir/Sub-Task4.exs
n = String.to_integer(File.read!("input.txt"))
File.open!("output.txt", [:write], fn file ->
  for i <- 0..(n-1) do
    IO.write(file, String.duplicate(" ", n-i-1) <> String.duplicate("*", 2*i + 1) <> "\n")
  end
  for i <- (n-2)..0 do
    IO.write(file, String.duplicate(" ", n-i-1) <> String.duplicate("*", 2*i + 1) <> "\n")
  end
end)
